legendKing.py

```diff
@@ -181,36 +216,146 @@ def setup_scale(vpName, maxValue, minValue, guide, reverse=None,
+            color2 = '#EDF700' # max extreme 
+            color1 = '#000078' # min extreme
+        
+        
+        if minValue*maxValue >= 0:
+            if reverse == True:
+                color2 = 'Grey80' #if span is all pos or all neg, and not reversed, set min extreme to grey80
+            else:
+                color1 = 'Grey80'
+            
+        if reverse: 
+            color1 , color2 = color2, color1
+            
+        # (TS) Reversed Rainbow corner-case
+        colormap_name = colormap 
+        if colormap == 'Rainbow' and reverse == True:
+            colormap_name = 'Reversed rainbow'
+        
+        
+        # (TS) Activate Colormap
+        contourOptions.setValues(
+                        spectrum=colormap_name, 
+                        outsideLimitsAboveColor=color2,
+                        outsideLimitsBelowColor=color1)
+        symbolOptions.setValues(
+                        tensorColorSpectrum=colormap_name,
+                        vectorColorSpectrum=colormap_name)
+                        
+
+# (TS)
+def create_cmap(colormap,reverse,N):
+    """generate the cmap according to the user-selection for non-Rainbow colors
+        > current implementation linearly interpolates between two bounding colors
+          with N spaces corresponding to calculated tick vector
+        > future development should 
+    """
+    
+    import abaqus
+    import numpy as np
+    "STEP 1: Determine bounding colors"
+    if colormap == 'Rainbow':
+        return
+    elif colormap == 'Symmetric':
+        color1_ext = (240,100,50) #00007F Darkest Blue
+        color1_mid = (220,40,100) #99BAFF Lightest Blue
+        color2_mid = (20, 40, 100)#FFBB99 Lightest red
+        color2_ext = (0, 100, 50) #7F0000 Darkest Red
+    # elif colormap == 'Cbs-cool':
+    #     color1 = (110, 10, 100)
+    #     color2 = (220, 90, 50)
+    
+    "STEP 2: Check if the color map needs reversed"
+    if (reverse == True) and colormap == 'Symmetric':
+        color1_ext, color2_ext = color2_ext, color1_ext #Darkest swap w/ Darkest
+        color1_mid, color2_mid = color2_mid, color1_mid #Lightest swap w/ Lightest
+
+    # elif (reverse == True) and colormap != 'Symmetric': #pull a quick swap there
+    #     color1, color2 = color2, color1
+    
+    
+    "STEP 3: Create interpolated list of hex colors between bounding colors"
+    if colormap == 'Symmetric':
+        bot_colors = generate_interpolated_hex_table(color1_ext,color1_mid,N)
+        top_colors = generate_interpolated_hex_table(color2_mid,color2_ext,N)
+        bot_colors.append('Grey80') #central pivot point should be grey
+        bot_colors.extend( top_colors ) #merge the two lists
+        
+        hex_colors = bot_colors
+    elif colormap in ['Cbs-cool', 'Cbs-warm']:   #self-created colormap tables
+        hex_colors = generate_interpolated_hex_table(color1, color2,N)
+    elif colormap in ['Viridis','Plasma']:
+        print('ARE WER HERE')
```

Comment:
Line 306 print-statement should be removed.

Reply:
You can remove things like that and push the changes up to your branch before it gets merged into the main project.